src/routes/_components/status/Status.html

```diff
@@ -31,7 +31,7 @@
     <StatusDetails {...params} />
   {/if}
   <StatusToolbar {...params} on:recalculateHeight />
-  {#if replyShown}
+  {#if replyShown && $isUserLoggedIn}
```

Comment:
Ideally we would disable the reply button if the user isn't logged in, or (even better) make it so that users can't browse timelines unless they're logged in.